### PR TITLE
play file updated

### LINKED ISSUE #6: Playing Game Page
This needs to be done by Sheetal.

### PR DESCRIPTION
**Pull request by Sheetal1012**

### CODE REVIEW

File: src/Components/Match/Play.js [hunk 1]
```diff
@@ -5,13 +5,36 @@ import './Play.css'
...
 class play extends Component {
   handleRESULT=()=>{this.props.history.push('/Result');}
-  state={ANames:[],Aplayers:[],Bplayers:[],BNames:[],score:0,total:0  }
-  heandlebatting=(e)=>{
-    const run = Math.floor(Math.random() *7);
-    this.setState({ score:run });
-  this.setState({total: this.state.total+run})
-}
-componentDidMount(){
+  state={ANames:[],Aplayers:[],Bplayers:[],BNames:[],scoreA:0,totalA:0,scoreB:0,totalB:0,incrementA:0,incrementB:0}
```
Comment (tarunkethwalia) [inline src/Components/Match/Play.js:8]: Proper indentation is required.

File: src/Components/Match/Play.js [hunk 2]
```diff
@@ -5,13 +5,36 @@ import './Play.css'
...
+  handlebattingA=()=>{
+    this.state.incrementA++
+     if(this.state.incrementA<=6){
+    
+      //const run = Math.floor(Math.random()*6);
+      //for (let i = 0; i <=6; i++) {
+      const runa = Math.floor(Math.random()*6);
```
Comment (tarunkethwalia) [inline src/Components/Match/Play.js:15]: Please make A and B capital in runa and runb

**Review by tarunkethwalia — CHANGES_REQUESTED** [review]
Please update the required changes.

**Comment by Richu98** [comment]
Branch name should be TOSSF-6 @Sheetal1012

**Pull request merged**